creduce/passes/abstract.py

```diff
@@ -0,0 +1,37 @@
+#TODO: Make argument optional
+
+import re
+import enum
+
+class AbstractPass:
+    @enum.unique
+    class Result(enum.Enum):
+        ok = 0
+        stop = 1
+        error = 2
```

Comment:
are python enums strongly typed? if not, it would make more sense to use larger numbers for these, that are unlikely to occur naturally